config/urls.py

```diff
@@ -22,6 +22,11 @@
         TemplateView.as_view(template_name="pages/sources.html"),
         name="sources",
     ),
+    path(
+        "results",
+        TemplateView.as_view(template_name="pages/results.html"),
+        name="results",
+    ),
```

Comment:
I think because this route deals with a page built from data, that it should be in the `judgments` app (in `urls.py` in that app). Add a dummy method to `views.py` to hook the route onto.

In urls.py:
`path("results", views.results, name="results"),`

In views.py:
```
def results():
    pass
```